### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by xandanman**

### CODE REVIEW

File: src/public/product.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    // if(!isset($_GET["id"])){
+    //     header("location: /");
+    //     exit();
+    // }
+
+    require_once $_SERVER['DOCUMENT_ROOT'] . '/config.php';
+    require_once LIB . '/util/util.php';
+    
+    $id=$_GET["id"];
```
[the remark this answers is not in the document]
Comment (xandanman) [reply src/public/product.php:11]: cedric ging da fice want een header werkt niet

File: src/public/product.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+</div>
+<div class = "flex">
+    <div>
+    <h2 class="card-title mt-8 mx-8"><?php echo $products["name"] ?></h2>
+    <div class="mx-8">  <?php echo $products["description"] ?></div>
+    </div>
+    <div>
+        <h2 class="card-title mt-8 mx-8"> about seller:</h2>
+    <div>
+        <div class="mt-4 mx-8"><?php echo $seller["firstname"]," ", $seller["lastname"] ?></div>
```
[the remark this answers is not in the document]
Comment (CedricVerlinden) [reply src/public/product.php:54]: Yes would be a good idea, only show full name to buyer when purchase is confirmed.

File: src/public/member/myProducts.php [hunk 3]
```diff
@@ -12,7 +12,7 @@
 $query = "SELECT * FROM products WHERE userid = ?";
 $data = fetch($query, ['type' => 'i', 'value' => $userid]);
 
-if (!isset($data["id"])) {
+if (!isset($userid)) {
```
[the remark this answers is not in the document]
Comment (CedricVerlinden) [reply src/public/member/myProducts.php:15]: Is already changed in my branch, it will override.

**Review by CedricVerlinden — APPROVED** [review]
I will make some code cleanup changes myself, great job!

Don't merge yet!

**Review by CedricVerlinden — APPROVED** [review]

**Pull request merged**